src/cmds/qsub.c

```diff
@@ -5273,60 +5327,60 @@ get_comm_filename(char *fl)
  * @param[in] server - Target server name of NULL in case of default
  *
  */
-void
+static void
 do_daemon_stuff(char *file, char *handle, char *server)
 {
-	HANDLE hPipe;
-	int rc, pipeRc;
-	HANDLE hEvent, hEventParent;
-	HANDLE hSockEvent;
-	OVERLAPPED oOverlap;
+	HANDLE h_pipe;
+	int rc, pipe_rc;
+	HANDLE h_event, h_event_parent;
+	HANDLE h_sock_event;
+	OVERLAPPED o_overlap;
 	int svr_sock = -1;
 	HANDLE handles[2];
 	time_t connect_time = 0;
 	time_t cred_connect_time = time(0); /* Record current time to compare against the credential timeout value of 30 mins */
 
 	sd_svr = -1; /* not connected */
-	hEventParent = atoi(handle);
-	if (hEventParent == -1)
+	h_event_parent = atoi(handle);
+	if (h_event_parent == -1)
 		goto error;
 
-	hEvent = CreateEvent(
+	h_event = CreateEvent(
 		NULL, /* default security attribute */
 		TRUE, /* manual-reset event */
 		FALSE, /* initial state = signaled */
 		NULL); /* unnamed event object */
-	if (hEvent == NULL)
+	if (h_event == NULL)
 		goto error;
 
-	hSockEvent = WSACreateEvent();
-	if (hSockEvent == NULL)
+	h_sock_event = WSACreateEvent();
+	if (h_sock_event == NULL)
 		goto error;
 
-	oOverlap.hEvent = hEvent;
-	hPipe = CreateNamedPipe(file,
+	o_overlap.h_event = h_event;
```

Comment:
You want:
o_overlap.hEvent = h_event;

Reply:
Thanks @mike0042 for catching this! I just pushed a commit to fix it.